grammars/php.cson

```diff
@@ -470,15 +470,8 @@
   'heredoc':
     'patterns': [
       {
-        'begin': '(?=<<<\\s*("?)([a-zA-Z_]+[a-zA-Z0-9_]*)(\\1)\\s*$)'
+        'begin': '(?=<<<\\s*("?)([a-zA-Z_]+[a-zA-Z0-9_]*)(\\1)[ \\t]*$)'
```

Comment:
according to the [original pull request](https://github.com/atom/language-php/pull/77/commits/eb4ae2656bd8a427d2026f082ba2c62da3d62654#diff-9972c3dda8fb0e2f637ca073c25e7e6dR439)
```
[ \\t] was used instead of \\s because \\s will match a newline
```